docs/dev/libvirt/load_balancer.md

```diff
@@ -0,0 +1,125 @@
+# Load Balancer Setup
+
+The libvirt deployment does not deploy a load balancer for development purposes. 
+
+This doc goes over an example configuration of HAProxy for doing local development.
+
+### Installing HAProxy
+These instructions are for Fedora 34+.
```

Comment:
Maybe word this as `These instructions were tested on Fedora 34`? Or do you expect that haproxy in RHEL8/Fedora 33 will be too old?

Reply:
I can change this to "tested" as that's more accurate. Have not tried in RHEL8 or 7.9.